README.md

```diff
@@ -27,7 +27,7 @@ And you can quit the Terminal.
 
 ### To quit/stop the Keylogger
 
-To quit the Keylogger, first find it's pid using `ps` and use `kill` to stop the keylogger.
+To quit the Keylogger, first, find it's PID using `ps` and use `kill` to stop the keylogger.
```

Comment:
`-it's`
`+its`

Reply:
The added comma not needed IMO.

Reply:
Thank you for notes! I think comma needed after "first".